AdminNotice.php

```diff
@@ -377,7 +382,10 @@ public static function fromJson($json) {
 				$notice->dismissible();
 			}
 			if ($properties['isPersistentlyDismissible']) {
-				$notice->persistentlyDismissible(self::getKey($properties, 'dismissionScope', self::DISMISS_PER_SITE));
+				$notice->persistentlyDismissible(
+					self::getKey($properties, 'dismissionScope', self::DISMISS_PER_SITE),
+					$properties['dismissalDuration']
```

Comment:
The `dismissalDuration` key might be missing, for example because the user upgraded from an older version of the library that didn't use this key. I recommend using getKey here to handle that possibility.
```suggestion
					self::getKey($properties, 'dismissalDuration', self::DISMISS_PERMANENTLY)
```